src/osyris/plot/parser.py

```diff
@@ -1,12 +1,14 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # Copyright (c) 2022 Osyris contributors (https://github.com/nvaytet/osyris)
 
-from matplotlib.colors import LogNorm, Normalize
+from matplotlib.colors import LogNorm, Normalize, SymLogNorm
 
 
 def get_norm(norm=None, vmin=None, vmax=None):
     if norm == "log":
         return LogNorm(vmin=vmin, vmax=vmax)
+    elif norm == "SymLog":
```

Comment:
Not sure what standard we want to use for these names.
Until now, I had everything as lowercase.
Maybe we should be slightly more flexible and allow any capitalization?
For example, something like `if norm.lower() == 'symlog'`?
And the same for `'log'`, so if you write `norm='Log'` it still works instead of silently returning a default linear normalization?